src/views/forms/FormAccountRestrictionTransaction/RestrictionFlagMapping.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 NEM (https://nem.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ *
+ */
+// @ts-ignore
+import { RestrictionDirection } from '@/components/RestrictionDirectionInput/RestrictionDirectionInput.vue';
+// @ts-ignore
+import { RestrictionBlockType } from '@/components/RestrictionTypeInput/RestrictionTypeInput.vue';
+import { AddressRestrictionFlag, MosaicRestrictionFlag, OperationRestrictionFlag } from 'symbol-sdk';
+import { AccountRestrictionTxType } from './FormAccountRestrictionTransactionTs';
+// @ts-ignore
+
+export default class RestrictionFlagMapping {
+    /**
+     * Maps parameters to RestrictionFlag
+     * @param restrictionTxType
+     * @param direction
+     * @param blockType
+     */
+    public static toRestrictionFlag(
+        restrictionTxType: AccountRestrictionTxType,
```

Comment:
here probably not to follow the SDKs enum types? just pass a number should be okay I guess. So it can use the base restriction flags + bitwise?

Reply:
Sorry, couldn't understand this. This is a mapping class, the idea is to convert to/from SDK enum types, right?